### ISSUE TITLE
java.lang.ClassCastException: com.rockerhieu.rvadapter.endless.EndlessRecyclerViewAdapter$PendingViewHolder cannot be cast to MyRecyclerViewAdapter$ViewHolder

### ISSUE THREAD

**Issue by devilmac** [rockerhieu/rv-adapter-endless#1]
Hi rockerhieu,

today I started to used your library and it works very well!
I use this endless adapter with a grid layout manager but, unfortunately, it got the error in title: this happens randomly, not always.
Below the stacktrace.

```
at com.rockerhieu.rvadapter.RecyclerViewAdapterWrapper.onBindViewHolder(RecyclerViewAdapterWrapper.java:48)
            at com.rockerhieu.rvadapter.endless.EndlessRecyclerViewAdapter.onBindViewHolder(EndlessRecyclerViewAdapter.java:89)
            at android.support.v7.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:5277)
            at android.support.v7.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:5310)
            at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4568)
            at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4461)
            at android.support.v7.widget.LayoutState.next(LayoutState.java:86)
            at android.support.v7.widget.StaggeredGridLayoutManager.fill(StaggeredGridLayoutManager.java:1423)
            at android.support.v7.widget.StaggeredGridLayoutManager.scrollBy(StaggeredGridLayoutManager.java:1926)
            at android.support.v7.widget.StaggeredGridLayoutManager.scrollVerticallyBy(StaggeredGridLayoutManager.java:1846)
            at android.support.v7.widget.RecyclerView$ViewFlinger.run(RecyclerView.java:3912)
            at android.view.Choreographer$CallbackRecord.run(Choreographer.java:767)
            at android.view.Choreographer.doCallbacks(Choreographer.java:580)
            at android.view.Choreographer.doFrame(Choreographer.java:549)
            at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
            at android.os.Handler.handleCallback(Handler.java:739)
            at android.os.Handler.dispatchMessage(Handler.java:95)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5254)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
```

**UPDATE 1:** for the moment I noticed this error only on Android 5.1.1, on lower version (i.e. Android 4.3) it seems to work without errors.

**UPDATE 2:** also in Android 4.3 the error is occurred.

**Comment by rockerhieu** [comment]
I know the problem, fixing...

**Comment by rockerhieu** [comment]
Fixed at 48ccf67d. I will release `1.1` soon.

**Comment by rockerhieu** [comment]
@devilmac can you try again with latest `1.1-SNAPSHOT`? If it is fixed then I will release `1.1` to maven.
